### PR TITLE
Added sets of hover& pressed tokens. Also updated warning/success/inverses 

### PR DESCRIPTION
**Pull request by Akatori-Design**
**Added**
|| LIGHT & DARK THEMES ||

Added `.default-hover` & `.default-pressed` colors added for `primary`, `success`, `error`, `warning` (but not for `info`)

```
- primary.default-hover
- primary.default-pressed

- success.default-hover
- success.default-pressed

- warning.default-hover
- warning.default-pressed

- error.default-hover
- error.default-pressed
```

**Edited**
|| LIGHT THEME ||
1. Edited `success.default` , `success.muted`, `warning.default`, `warning.muted` to be more accessible 
2. Edited `Inverse `colors from `#FCFCFC` to `#FFFFFF` (white000)

```
- success.default
- success.muted

- warning.default
- warning.muted

- primary.inverse
- error.inverse
- success.inverse
- warning.inverse
- info.inverse
```
|| DARK THEME ||

1. Edited `error.default`, `error.muted` for accessibility.
2. Edited `Inverse` colors from `#FCFCFC` to `#141618` (grey900)

```
- error.default
- error.muted

- primary.inverse
- error.inverse
- success.inverse
- warning.inverse
- info.inverse
```

Fixes: #580

### CODE REVIEW

File: src/figma/tokens.json [hunk 1]
```diff
@@ -166,8 +166,8 @@
           "type": "color"
         },
         "orange600": {
-          "value": "#C65507",
-          "description": "(HEX: #C65507)",
+          "value": "#BF5208",
+          "description": "(HEX: #BF5208)",
           "type": "color"
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:171]: Screenshot 2024-01-18 at 5 30 48 PM

Screenshot 2024-01-18 at 5 24 35 PM

File: src/figma/tokens.json [hunk 2]
```diff
@@ -218,8 +218,8 @@
...
         },
         "green600": {
-          "value": "#1E7E34",
-          "description": "(HEX: #1E7E34)",
+          "value": "#1C8234",
+          "description": "(HEX: #1C8234)",
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:222]: Screenshot 2024-01-18 at 5 31 13 PM

Screenshot 2024-01-18 at 5 23 58 PM

File: src/figma/tokens.json [hunk 3]
```diff
@@ -260,8 +260,8 @@
           "type": "color"
         },
         "red400": {
-          "value": "#E06470",
-          "description": "(HEX: #E06470)",
+          "value": "#FF5263",
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:263]: ### Before

Screenshot 2024-01-18 at 5 31 36 PM

### After 

Screenshot 2024-01-18 at 5 25 05 PM

File: src/figma/tokens.json [hunk 4]
```diff
@@ -943,14 +943,24 @@
           "type": "color"
         },
         "inverse": {
-          "value": "#FCFCFC",
-          "description": "(white010: #FCFCFC) For elements used on top of primary/default. (Example: label of primary button, check in a checkbox)",
-          "type": "color"
+          "value": "#FFFFFF",
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:946]: ### Before
Screenshot 2024-01-18 at 5 32 16 PM


### After

Screenshot 2024-01-18 at 5 25 49 PM

File: src/figma/tokens.json [hunk 5]
```diff
@@ -943,14 +943,24 @@
...
+        },
+        "default-hover": {
+          "value": "#0379CE",
+          "type": "color",
+          "description": "(#0379CE) For the \"hover\" state of interactive primary elements"
+        },
+        "default-pressed": {
+          "value": "#036DB9",
+          "type": "color",
+          "description": "(#036DB9) For the \"pressed\" state of interactive primary elements"
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:963]: ### Before
Screenshot 2024-01-18 at 5 32 16 PM


### After

Screenshot 2024-01-18 at 5 25 49 PM

File: src/figma/tokens.json [hunk 6]
```diff
@@ -960,8 +970,8 @@
           "type": "color"
         },
         "alternative": {
-          "value": "#C65507",
-          "description": "(orange600: #C65507) [DEPRECATED] secondary.alternative should be used as an alternative to secondary.default for things such as hover states",
+          "value": "#BF5208",
+          "description": "(orange600: #BF5208) [DEPRECATED] secondary.alternative should be used as an alternative to secondary.default for things such as hover states",
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:974]: This is the update for `orange600` find and replace #C65507 => #BF5208

File: src/figma/tokens.json [hunk 7]
```diff
@@ -997,68 +1007,98 @@
...
+        },
+        "default-hover": {
+          "value": "#CD4250",
+          "type": "color",
+          "description": "(#CD4250) For the \"hovered\" state of interactive danger/critical elements"
+        },
+        "default-pressed": {
+          "value": "#C63441",
+          "type": "color",
+          "description": "(#C63441) For the \"pressed\" state of interactive danger/critical elements"
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:1027]: ### Before

Screenshot 2024-01-18 at 5 31 36 PM


### After

Screenshot 2024-01-18 at 5 26 24 PM

File: src/figma/tokens.json [hunk 8]
```diff
@@ -997,68 +1007,98 @@
...
+          "type": "color",
+          "description": "(orange600: #BF5208 10% opacity) For lowest contrast background used in warning elements. (Example: notification background)"
         },
         "inverse": {
-          "value": "#FCFCFC",
-          "description": "(white010: #FCFCFC) For elements used on top of warning/default. Used for text, icon or border",
-          "type": "color"
+          "value": "#FFFFFF",
+          "type": "color",
+          "description": "(white000: #FFFFFF) For elements used on top of warning/default. Used for text, icon or border"
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:1049]: ### Before

Screenshot 2024-01-18 at 5 32 55 PM


### After

Screenshot 2024-01-18 at 5 26 51 PM

File: src/figma/tokens.json [hunk 9]
```diff
@@ -997,68 +1007,98 @@
...
+        },
+        "default-hover": {
+          "value": "#C2540A",
+          "type": "color",
+          "description": "(#C2540A) For the \"hover\" state of warning elements"
+        },
+        "default-pressed": {
+          "value": "#A24507",
+          "type": "color",
+          "description": "(#A24507) For the \"pressed\" state of warning elements"
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:1064]: ### Before

Screenshot 2024-01-18 at 5 32 55 PM


### After

Screenshot 2024-01-18 at 5 26 51 PM

File: src/figma/tokens.json [hunk 10]
```diff
@@ -997,68 +1007,98 @@
...
+        },
+        "default-hover": {
+          "value": "#208838",
+          "type": "color",
+          "description": "(#208838) For the \"hover\" state of success elements"
+        },
+        "default-pressed": {
+          "value": "#1B7431",
+          "type": "color",
+          "description": "(#1B7431) For the \"pressed\" state of success elements"
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:1101]: ### Before
Screenshot 2024-01-18 at 5 33 12 PM


### After

Screenshot 2024-01-18 at 5 27 20 PM

File: src/figma/tokens.json [hunk 11]
```diff
@@ -1078,9 +1118,9 @@
           "type": "color"
         },
         "inverse": {
-          "value": "#FCFCFC",
-          "description": "(white010: #FCFCFC) For elements used on top of info/default. Used for text, icon or border",
-          "type": "color"
+          "value": "#FFFFFF",
+          "type": "color",
+          "description": "(white000: #FFFFFF) For elements used on top of info/default. Used for text, icon or border"
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:1123]: Update to inverse for info

File: src/figma/tokens.json [hunk 12]
```diff
@@ -1338,14 +1378,24 @@
...
+        },
+        "default-hover": {
+          "value": "#0092FA",
+          "type": "color",
+          "description": "(#0092FA) For the \"hover\" state of interactive primary elements"
+        },
+        "default-pressed": {
+          "value": "#54B6FC",
+          "type": "color",
+          "description": "(#54B6FC) For the \"hover\" state of interactive primary elements"
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:1398]: ### Before

Screenshot 2024-01-18 at 5 33 38 PM


### After

Screenshot 2024-01-18 at 5 27 52 PM

File: src/figma/tokens.json [hunk 13]
```diff
@@ -1377,29 +1427,39 @@
...
+          "type": "color",
+          "description": "(#FF5263) 15% opacity) For lowest contrast background used in high-level alert danger/critical elements. (Example: notification background)"
         },
         "inverse": {
-          "value": "#FCFCFC",
-          "description": "(white010: #FCFCFC) For elements used on top of error/default (Example: label of danger/critical button)",
-          "type": "color"
+          "value": "#141618",
+          "type": "color",
+          "description": "(grey900: #141618) For elements used on top of error/default (Example: label of danger/critical button)"
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:1447]: ### Before

Screenshot 2024-01-18 at 5 34 02 PM


### After

Screenshot 2024-01-18 at 5 28 29 PM

File: src/figma/tokens.json [hunk 14]
```diff
@@ -1377,29 +1427,39 @@
...
+        },
+        "default-hover": {
+          "value": "#FF4D58",
+          "type": "color",
+          "description": "FF4D58: For the \"hover\" state of interactive danger/critical elements"
+        },
+        "default-pressed": {
+          "value": "#F9868E",
+          "type": "color",
+          "description": "F9868E: For the \"hover\" state of interactive danger/critical elements"
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:1462]: See above screenshot

File: src/figma/tokens.json [hunk 15]
```diff
@@ -1427,6 +1487,16 @@
...
+        },
+        "default-hover": {
+          "value": "#FFC60A",
+          "type": "color",
+          "description": "(#FFC60A): For warning with stronger contrast."
+        },
+        "default-pressed": {
+          "value": "#FFEAA3D1",
+          "type": "color",
+          "description": "(#FFEAA3): For the \"pressed\" state of low-mid level alert elements."
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:1499]: ### Before

Screenshot 2024-01-18 at 5 34 26 PM


### After
Screenshot 2024-01-18 at 5 29 00 PM

File: src/figma/tokens.json [hunk 16]
```diff
@@ -1446,14 +1516,24 @@
...
+        },
+        "default-hover": {
+          "value": "#05B82F",
+          "type": "color",
+          "description": "(#05B82F): For success with stronger contrast."
+        },
+        "default-pressed": {
+          "value": "#6AEC88",
+          "type": "color",
+          "description": "(#6AEC88): For success with stronger contrast."
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:1536]: ### Before

Screenshot 2024-01-18 at 5 34 48 PM


### After

Screenshot 2024-01-18 at 5 29 31 PM

File: src/figma/tokens.json [hunk 17]
```diff
@@ -1473,9 +1553,9 @@
           "type": "color"
         },
         "inverse": {
-          "value": "#FCFCFC",
-          "description": "(white010: #FCFCFC) For elements used on top of info/default. Used for text, icon or border",
-          "type": "color"
+          "value": "#141618",
+          "type": "color",
+          "description": "(grey900: #141618) For elements used on top of info/default. Used for text, icon or border"
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:1558]: See screenshot above for success inverse update

File: src/js/brandColors/brandColors.test.ts [hunk 18]
```diff
@@ -7,403 +7,403 @@ describe('Brand Colors', () => {
   describe('White', () => {
     it('js tokens for white000 matches figma tokens white000', () => {
       expect(brandColors.white['000']).toStrictEqual(
-        designTokens.global.brandColors.white.white000.value.toLowerCase(),
+        designTokens.global.brandColors.white.white000.value,
```
Comment (georgewrmarshall) [inline src/js/brandColors/brandColors.test.ts:10]: Now that the brand color hex values are all uppercase there is no need for this toLowerCase method

File: src/js/brandColors/brandColors.ts [hunk 19]
```diff
@@ -2,87 +2,87 @@ import { BrandColors } from './brandColors.types';
 
 export const brandColors: BrandColors = {
   white: {
```
Comment (georgewrmarshall) [inline src/js/brandColors/brandColors.ts:4]: Updating casing to uppercase to match with the rest of the javascript color values

File: src/js/themes/darkTheme/colors.ts [hunk 20]
```diff
@@ -39,9 +39,11 @@ export const colors: ThemeColors = {
   },
   primary: {
     default: '#1098FC',
+    defaultHover: '#0092FA',
```
Comment (georgewrmarshall) [inline src/js/themes/darkTheme/colors.ts:42]: Matches figma and has css in js and css variable token
Screenshot 2024-01-18 at 5 56 07 PM

File: src/js/themes/darkTheme/colors.ts [hunk 21]
```diff
@@ -39,9 +39,11 @@ export const colors: ThemeColors = {
...
+    defaultPressed: '#54B6FC',
```
Comment (georgewrmarshall) [inline src/js/themes/darkTheme/colors.ts:43]: Matches figma and has css in js and css variable token

Screenshot 2024-01-18 at 5 57 00 PM

File: src/js/themes/darkTheme/colors.ts [hunk 22]
```diff
@@ -39,9 +39,11 @@ export const colors: ThemeColors = {
...
     alternative: '#43AEFC',
     muted: '#1098FC26',
-    inverse: '#FCFCFC',
+    inverse: '#141618',
```
Comment (georgewrmarshall) [inline src/js/themes/darkTheme/colors.ts:46]: Matches figma and has css in js and css variable token

Screenshot 2024-01-18 at 5 58 25 PM

File: src/js/themes/darkTheme/colors.ts [hunk 23]
```diff
@@ -54,31 +56,37 @@ export const colors: ThemeColors = {
   },
   error: {
     default: '#D73847',
-    alternative: '#E06470',
+    defaultHover: '#FF4D58',
```
Comment (georgewrmarshall) [inline src/js/themes/darkTheme/colors.ts:59]: Matches figma and has css in js and css variable token

Screenshot 2024-01-18 at 5 58 57 PM

File: src/js/themes/darkTheme/colors.ts [hunk 24]
```diff
@@ -54,31 +56,37 @@ export const colors: ThemeColors = {
...
+    defaultPressed: '#F9868E',
```
Comment (georgewrmarshall) [inline src/js/themes/darkTheme/colors.ts:60]: Matches figma and has css in js and css variable token

Screenshot 2024-01-18 at 6 00 11 PM

File: src/figma/tokens.json [hunk 25]
```diff
@@ -1427,6 +1487,16 @@
...
           "description": "(yellow500: #FFD33D 50% opacity) [DEPRECATED] warning.disabled should be used for all disabled component colors such as buttons",
           "type": "color"
+        },
+        "default-hover": {
+          "value": "#FFC60A",
+          "type": "color",
+          "description": "(#FFC60A): For warning with stronger contrast."
+        },
+        "default-pressed": {
+          "value": "#FFEAA3D1",
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:1497]: Remove the opacity from the hex
```suggestion
          "value": "#FFEAA3",
```

File: src/figma/tokens.json (hunk 3 above)
Comment (georgewrmarshall) [inline src/figma/tokens.json:263]: This change should be reverted
```suggestion
          "value": "#E06470",
```

File: src/figma/tokens.json [hunk 26]
```diff
@@ -1377,29 +1427,39 @@
...
         "default": {
-          "value": "#D73847",
-          "description": "(red500: #D73847) For high-level alert danger/critical elements. Used for text, background, icon or border",
-          "type": "color"
+          "value": "#FF5263",
+          "type": "color",
+          "description": "(#FF5263) For high-level alert danger/critical elements. Used for text, background, icon or border"
         },
         "alternative": {
           "value": "#E06470",
```
Comment (georgewrmarshall) [inline src/figma/tokens.json:1435]: Update to `red300`

**Review by brianacnguyen — APPROVED** [review]

**Pull request merged**